crates/air-lib/interpreter-data/src/executed_state/se_de.rs

```diff
@@ -96,19 +100,23 @@ pub mod sender_serializer {
             }
 
             fn visit_str<E: serde::de::Error>(self, raw_sender: &str) -> Result<Self::Value, E> {
```

Comment:
I think we can remove this custom se/de, it was introduced to prevent breaking change in AVM data when `PeerWithId` was introduced.